panel/src/components/Forms/Toolbar/LinkDialog.vue

```diff
@@ -65,6 +61,28 @@ export default {
 				return `<${this.value.url}>`;
 			}
 		},
+		open(input, selection) {
+			this.parseDOM(selection);
+			this.$refs.dialog.open();
+		},
+		parseDOM(selection) {
```

Comment:
One last comment I have is the name: is this really DOM-related? Would think more like “parseSelection”

Reply:
Actually I inspired from writer nodes 🙈 and that's really a node in there. But this is just string selection as your said. You're right 👍